portal-impl/src/com/liferay/mail/internal/MailEngine.java

```diff
@@ -511,7 +506,7 @@ private static void _validateAddresses(Address[] addresses)
 
 	private static final String _TEXT_PLAIN = "text/plain;charset=\"UTF-8\"";
 
-	private static final Log _log = LogFactoryUtil.getLog(MailEngine.class);
+	private static final Log _log = LogFactoryUtil.getLog(MailServiceUtil.class);
```

Comment:
@julschong You should not change the class name, the class name here should be consistent with the class who defines it.